lib/yamatanooroti/vterm.rb

```diff
@@ -4,7 +4,8 @@
 require 'io/console'
 
```

Comment:
It seems to ```require 'io/wait'``` required for RUBY_VERSION <3.2.0

Reply:
Thank you :+1: added